### PR TITLE
Fixed Form 2.7 compatibility in type and crud generation

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: Resources/skeleton/crud/actions/delete.php.twig [hunk 1]
```diff
@@ -22,7 +22,7 @@
         if ($form->isSubmitted() && $form->isValid()) {
             $em = $this->getDoctrine()->getManager();
             $em->remove(${{ entity_singularized }});
-            $em->flush();
+            $em->flush(${{ entity_singularized }});
```
Comment (javiereguiluz) [inline Resources/skeleton/crud/actions/delete.php.twig]: This change looks unrelated 😉

**Comment by fabpot** [comment]
Thank you @HeahDude.
